modules/virt-vm-wizard-fields-web.adoc

```diff
@@ -89,6 +93,6 @@ endif::[]
 ifdef::virtualmachine[]
 |Start this virtual machine after creation.
 |
-|Select to automatically start the virtual machine upon creation.
+|This checkbox is selected by default and the virtual machine starts running after creation. Clear the checkbox if you do not want the virtual machine to start when it is created.
```

Comment:
Suggestion: Checkbox to indicate if the virtual machine should start after it is created. It is selected by default.

Reply:
In referring to the IBM Style Guide, they recommend "clear the checkbox" - which is the usage in this case. Here I am describing the default behavior first followed by how to undo the default behavior. Prefer this approach but I appreciate your suggestion.first